deploy/mlflow-triton-plugin/setup.py

```diff
@@ -32,6 +32,6 @@
     long_description=open("README.md").read(),
     long_description_content_type="text/markdown",
     packages=find_packages(),
-    install_requires=["mlflow>=2.2.1,<3.0", "tritonclient[all]", "boto3"],
+    install_requires=["mlflow>=2.2.1,<3.0", "tritonclient[all]", "boto3", "aiohttp==3.8.4"],
```

Comment:
The `aiohttp` requirement is set in the client [requirements_http.txt](https://github.com/triton-inference-server/client/blob/main/src/python/library/requirements/requirements_http.txt#L28). Could you make the change over there?

I think it would be better to allow a small range of acceptable version, like `aiohttp>=3.8.1,<4.0.0`.

Reply:
Please see the new one (https://github.com/triton-inference-server/client/pull/297) on `client` side. Let's close this one.

Can a new version of tritonclient be released to pypi soon? I'd like to use `mlflow-triton` in Python 3.10. Thanks.